schema/me/conversations.js

```diff
@@ -0,0 +1,73 @@
+import date from '../fields/date';
+import impulse from '../../lib/loaders/impulse';
+import gravity from '../../lib/loaders/gravity';
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLList,
+  GraphQLInt,
+} from 'graphql';
+const { IMPULSE_APPLICATION_ID } = process.env;
+
+const ConversationType = new GraphQLObjectType({
+  name: 'ConversationType',
+  description: 'A conversation.',
+  fields: {
```

Comment:
These aren't all the fields, just some ones to start. I sort of arbitrarily picked them, I will double check the conversations schema (and also maybe try to be more typed instead of just strings!).